src/pepr/operator/controllers/network/policies.ts

```diff
@@ -59,36 +91,54 @@ export async function networkPolicies(pkg: UDSPackage, namespace: string, istioM
   const exposeList = pkg.spec?.network?.expose ?? [];
   // Iterate over each exposed service, excluding directResponse services
   for (const expose of exposeList.filter(exp => !exp.advancedHTTP?.directResponse)) {
-    const { gateway = Gateway.Tenant, port, selector = {}, targetPort } = expose;
-
-    // Use the same port as the VirtualService if targetPort is not set
+    const { gateway = Gateway.Tenant, port = 80, selector = {}, targetPort } = expose;
```

Comment:
Was the default addition of `80` intentional here? It _shouldn't_ ever get hit so I think it might just cause confusion.